packages/components/src/List/Toolbar/Toolbar.component.js

```diff
@@ -81,60 +79,47 @@ function Toolbar({
 		};
 	}
 	const displayModeId = id && `${id}-display-mode`;
-	const hasToolbarItem = selectAllCheckbox || display || sort || pagination || filter;
+	const showSelectAll = selectAllCheckbox && display && display.mode === 'large';
+	const showSort = sort && display && display.mode === 'large';
```

Comment:
Those 2 are already managed in List.component.js. No need to redefine the conditions here.
The toolbar just displays things based on what is passed as props, and the List just pass things or not with more logic.

Reply:
Still ko. Remove this showSort, `props.sort` already passed depending on this condition in List.component